notebooks/official/ml_metadata/vertex-pipelines-ml-metadata.ipynb

```diff
@@ -0,0 +1,1179 @@
+{
```

Comment:
<p>Again, suggest explaining the steps:</p><p>1. Import CSV data into DataFrame</p><p>2. Extract and remove the label column</p><p>3. Split data into train and test data.</p><p>4. Train the model.</p><p>5. Log the metrics.</p><p>6. Save the model artifacts.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>